__tests__/app.test.js

```diff
@@ -117,4 +117,42 @@ describe("GET /api/articles", () => {
       expect(articles[0].comment_count).toEqual("2")
     })
   })
+})
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the given article_id with the correct properties, sorted by date descending", () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({ body: { comments } }) => {
+      expect(comments.length).toBe(11)
+      comments.forEach((comment) => {
+        expect(comment).toMatchObject({
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          article_id: expect.any(Number)
```

Comment:
You are not expecting any article_id here, you are expecting article_id 1 as that is the one requested in the endpoint